packages/codemod/src/transformations/refine3-to-refine4.ts

```diff
@@ -0,0 +1,77 @@
+import { API, JSCodeshift, Collection, FileInfo } from "jscodeshift";
+
+export const parser = "tsx";
+
+const REFINE_ANTD_PATH = "@pankod/refine-antd";
+const REFINE_CORE_PATH = "@pankod/refine-core";
+
+const ANTD_IMPORTS_TO_MOVE_CORE = ["useMenu"];
```

Comment:
```suggestion
const UI_IMPORTS_TO_MOVE_CORE = ["useMenu"];
```

I think we can do better with the naming here 😅